released/packages/coq-coqprime-generator/coq-coqprime-generator.1.1.2/opam

```diff
@@ -31,7 +31,7 @@ install: [
   [make "install"]
 ]
 depends: [
-  "ocaml" {>= "5.0"}
+  "ocaml" {>= "4.14.2"}
```

Comment:
As far as I can tell, the only change to the OCaml code was to change the dependency from `num` to `zarith`. Since the previous version was working with any version of OCaml, the new version will work with any version of OCaml (that supports Zarith). So,
```suggestion
  "ocaml"
```